assets/prefabs/furnishings/chest.prefab

```diff
@@ -1,4 +1,7 @@
 {
+    "DisplayName": {
+        "name": "${coreadvancedassets:game#furnishings-chest}"
```

Comment:
One drawback of this approach is that if a screen just renders this value as a text without an explicit call to `TranslationSystem#translate(...)` this unreadable string will show up...

I don't understand (yet) why using this reference string in a `.ui` file directly is automatically translated, but we return this string in a programmatic binding it is not. :thinking: 

See https://github.com/Terasology/Inventory/pull/45/files#diff-04de2f5098d10bdd839f6f8c99d30807432d03ca23c6eebe36ab5a715341109bR11